helpers/orderBy.js

```diff
@@ -77,4 +77,29 @@ function OrderBy (orderBy, options) {
 	};
 }
 
+/**
+ * Converts orderBy queries from string format like "field:asc"
+ * to format accepted by "json-sql" library: {field: 1}.
+ * Ascending sort method number equivalent is 1.
+ * Descending sort method number equivalent is -1.
+ * @param {string} sortQuery
+ * @param {Array} sortableFields
+ * @returns {Object}[={}] returns {} if incorrect format of sortQuery given or if field
+ */
+OrderBy.sortQueryToJsonSqlFormat = function (sortQuery, sortableFields) {
+	var sortQueryMatched = typeof sortQuery !== 'string' ? null : sortQuery.match(/^([a-zA-Z0-9]+):(asc|desc)$/);
```

Comment:
The regular expression here is based on assumption that fields will never contain any non-alphanumeric letter. But its common to have `_` in field names. Is this assumption is documented some where, so no one in future may created such field names.

Reply:
Relying on Swagger documentation there are no fields containing `_`.

Reply:
But as we introduce new fields in swagger,  when we are not sure of underlying code, code may break. Either we should include your regular expression into Swagger documentation, so every one be sure of when introducing new fields. Or we should make this regular expression as flexible. I think we should go for second option.